libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -1983,6 +1983,8 @@ const char * IotMqtt_OperationType( IotMqttOperationType_t operation )
 /*-----------------------------------------------------------*/
 
 /* Provide access to internal functions and variables if testing. */
+/* IOT_BUILD_TESTS is defined outside the codebase, e.g. passed in by CMake command. */
+/* coverity[misra_c_2012_rule_20_9_violation] */
```

Comment:
I believe this also needs a `/* coverity[caretline] */` to suppress this specific violation.

Reply:
You are right. Fixed it.